src/modules/Channel/context/ChannelProvider.tsx

```diff
@@ -319,7 +321,7 @@ const ChannelProvider: React.FC<ChannelContextProps> = (props: ChannelContextPro
   // this hook sets currentGroupChannel asynchronously
   useGetChannel(
     { channelUrl, sdkInit, disableMarkAsRead },
-    { messagesDispatcher, sdk, logger, markAsReadScheduler },
+    { messagesDispatcher, sdk, logger, markAsReadScheduler, eventHandlers },
```

Comment:
Maybe I'm mistaken, but it looks like `eventHandlers` is just being passed down to `useGetChannel` hook, but it's fetched from `const globalStore = useSendbirdStateContext();`. Can't we just `useSendbirdStateContext()` directly from `useGetChannel`?

Reply:
Yes! We could. I was trying to match the logger pattern in this instance and they seem to pass it in.